src/react/canfar/CanfarLoginModal.js

```diff
@@ -75,12 +77,12 @@ class CanfarLoginModal extends React.Component {
                          placeholder="Password"/>
                 </div>
                 {errMsg}
-                <a href={this.state.modalURLs.passreset}
+                <a href={this.state.modalURLs[PASS_RESET_URI]}
```

Comment:
Is `passreset` still required in `HEADER_URL_DEFAULTS`, if it's no longer referenced in this location?

I have the same question for `acctrequest`, `groups`, and `search`.

Reply:
Ah, good catch.  I've updated those to match the URIs.